pkg/api/kubernetesagentpool/v20170727/types.go

```diff
@@ -0,0 +1,114 @@
+package v20170727
```

Comment:
What is the difference between: pkg/api/kubernetesagentpool/types.go  and pkg/api/kubernetesagentpool/v20170727/types.go? Why does agent pool only have another API version? The only publicly exposed API version should be 20170831 that I added to expose HostedMaster type.